### ISSUE TITLE
from pb tf_dst_dtype == DT_UINT8 || tf_dst_dtype == DT_INT32 || tf_dst_dtype == DT_FLOAT  Abort trap: 6

### ISSUE THREAD

**Issue by zyxcambridge** [tensorflow/tensorflow#15387]
cambridgedeMBP:tensorflow-master cambridge$ bazel-bin/tensorflow/contrib/lite/toco/toco \
>   --input_file=/Users/cambridge/Desktop/test/my_inception_v4_freeze.pb \
>   --output_file=/Users/cambridge/Desktop/test/my_inception_v4_freeze.pb.lite \
>   --input_format=TENSORFLOW_GRAPHDEF \
>   --output_format=TFLITE \
>   --inference_type=FLOAT \
>   --input_shapes=1,299,299,3 \
>   --input_arrays=input \
>   --output_array=InceptionV4/Logits/Predictions
2017-12-15 16:18:25.486891: I tensorflow/contrib/lite/toco/import_tensorflow.cc:140] Unsupported data type in placehoder op: 7
2017-12-15 16:18:25.487509: I tensorflow/contrib/lite/toco/import_tensorflow.cc:1046] Converting unsupported operation: DecodeJpeg
2017-12-15 16:18:25.487725: I tensorflow/contrib/lite/toco/import_tensorflow.cc:1046] Converting unsupported operation: StridedSlice
2017-12-15 16:18:25.487772: F tensorflow/contrib/lite/toco/import_tensorflow.cc:1160] Check failed: tf_dst_dtype == DT_UINT8 || tf_dst_dtype == DT_INT32 || tf_dst_dtype == DT_FLOAT 
Abort trap: 6

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue? Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tatatodd** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
